### ISSUE TITLE
Compliation of Outdated or Deprecated WCAG 2 Resouces

### ISSUE THREAD

**Issue by a11ydoer** [w3c/wcag#3014]
We will use this issue to complie outdated or deprecated WCAG 2 resources so that Working Group members can work those later when WCAG 3 migration happens.

**Comment by fstrr** [comment]
What's being classed as a "resource"? Do you mean things like really outdated Techniques? If so, I'd like to get removed long before WCAG 3. If you don't mean Techniques, I'll start a separate thread for those as there are quite a few.

**Comment by fstrr** [comment]
[C18: Using CSS margin and padding rules instead of spacer images for layout design](https://www.w3.org/WAI/WCAG21/Techniques/css/C18)

We don't need this any more, I think

**Comment by fstrr** [comment]
[C19: Specifying alignment either to the left OR right in CSS](https://www.w3.org/WAI/WCAG21/Techniques/css/C19)

This is a technique to use either `text-align:left` or `text-align:right` to align text. We don't need this.

**Comment by fstrr** [comment]
[SCR50: Providing a mechanism to allow users to display moving, scrolling, or auto-updating text in a static window or area](https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR36) and related [SCR50 working example](https://www.w3.org/WAI/WCAG21/working-examples/script-scroll-text-expand/).

I don't think I've ever seen anything like this in the wild.

**Comment by fstrr** [comment]
[C29: Using a style switcher to provide a conforming alternate version](https://www.w3.org/WAI/WCAG21/Techniques/css/C29)

We don't need this as it is at the moment, but could it be re-written so it's relevant to dark modes?

**Comment by fstrr** [comment]
[C23: Specifying text and background colors of secondary content such as banners, features and navigation in CSS while not specifying text and background colors of the main content](https://www.w3.org/WAI/WCAG21/Techniques/css/C23)

We don't need this as it is, but could it be replaced with an article on [forced colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors)?

**Comment by fstrr** [comment]
[C30: Using CSS to replace text with images of text and providing user interface controls to switch](https://www.w3.org/WAI/WCAG21/Techniques/css/C30).

I don't think anyone uses this technique anymore, and it's not something we should encourage.

**Comment by fstrr** [comment]
[G146: Using Liquid Layout](https://www.w3.org/WAI/WCAG21/Techniques/general/G146).

This uses floats to create three percentage-based columns. It's not responsive. This could do with a re-write to get it up to date. Oddly, the word "responsive" doesn't appear in the [index page of WCAG 2.1 Techniques](https://www.w3.org/WAI/WCAG21/Techniques/).

**Comment by fstrr** [comment]
The [SMIL Techniques](https://www.w3.org/WAI/WCAG21/Techniques/#smil) need to either be updated or removed. Does anyone use SMIL at all? I note that [the synchronized multimedia working group was closed in 2012](https://www.w3.org/AudioVideo/).

**Comment by fstrr** [comment]
Someone who knows about server-side scripting should audit the [server-side techniques](https://www.w3.org/WAI/WCAG21/Techniques/#server-side-script). There's only five of them, but going to assume they've not been updated since WCAG 2.0.

**Comment by momdo** [comment]
@fstrr 

EPUB 3.3 refers to SMIL 3.0 as Normative references.
https://www.w3.org/TR/epub-33/#bib-smil3

We suppose that SMIL3 supersede [SMIL2](https://www.w3.org/TR/SMIL2/) and [SMIL1](https://www.w3.org/TR/SMIL1/), but they have been valid W3C Recommendations. I think that these specs should be retired according to the [W3C Process Document](https://www.w3.org/2021/Process-20211102/#tr-end).

FYI: I raised https://github.com/w3c/wcag/issues/225 7 years ago...
